src/components/Modal/Modal.module.scss

```diff
@@ -15,7 +15,7 @@
   }
 }
 
-:global([data-reach-dialog-overlay]) {
```

Comment:
this doesn't seem to impact the components themselves, but was there a reason we added this for users of `palmetto-components`?

Reply:
I believe the components will throw a warning if the reach classes aren't included.

Reply:
not 100% sure but because of css modules the class names get mangled and thus the warning is thrown.

Reply:
this change doesn't generate any new warnings. there's another line in the css that you commented addresses this: https://github.com/palmetto/palmetto-components/blob/main/src/components/Modal/Modal.module.scss#L3-L4

Reply:
aah yes, forgot about that one. I think we could've removed the global classes at that point too.